StreamChatClient_v3/ChatClient.swift

```diff
@@ -0,0 +1,85 @@
+//
+// ChatClient.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+import Foundation
+
+public final class ChatClient {
+  // MARK: - Public
+
+  public let currentUser: User
+
+  public let config: Config
+
+  public convenience init(currentUser: User, config: ChatClient.Config = .init()) {
+    // All production workers
+    let workers: [WorkerBuilder] = [
+      MessageSender.init
+    ]
+
+    self.init(
+      currentUser: currentUser,
+      config: config,
+      workers: workers,
```

Comment:
This is `workerBuilders` not just workers. Just below you use `workers` as `[Worker]`.